website/package.json

```diff
@@ -12,12 +12,10 @@
     "clear": "docusaurus clear"
   },
   "dependencies": {
-    "@docusaurus/core": "2.0.0-alpha.70",
-    "@docusaurus/preset-classic": "2.0.0-alpha.70",
-    "@mdx-js/react": "^1.6.21",
-    "clsx": "^1.1.1",
-    "react": "^16.8.4",
-    "react-dom": "^16.8.4"
+    "@docusaurus/core": "2.4.3",
+    "@docusaurus/preset-classic": "2.4.3",
+    "react": "17.0.2",
+    "react-dom": "17.0.2"
```

Comment:
can't we go higher here? (react18)

Reply:
I tried react 18 but got a bunch of warnings that it isn't supported.